ibis/expr/window.py

```diff
@@ -228,10 +274,43 @@ def trailing_window(periods, group_by=None, order_by=None):
     -------
     win : ibis Window
     """
-    return Window(preceding=periods, following=0,
+    return Window(preceding=rows, following=0,
                   group_by=group_by, order_by=order_by)
 
 
+def trailing_time_window(prior_interval, order_by, group_by=None):
+    """
+    Create a trailing time window for use with aggregate window functions.
+
+    Parameters
+    ----------
+    prior_interval : TODO
```

Comment:
How do you feel about calling this `preceding` to be consistent with the other window creation functions?

Reply:
Sounds good.